psalm-baseline.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="5.3.0@b6faa3e96b8eb50ec71384c53799b8a107236bb6">
+  <file src="test/unit/Changelog/ChangelogReleaseNotesTest.php">
+    <UnusedMethodCall occurrences="1">
+      <code>setAccessible</code>
+    </UnusedMethodCall>
+  </file>
+  <file src="test/unit/Github/Api/GraphQL/Query/GetMilestoneChangelog/Response/LabelTest.php">
+    <InvalidArgument occurrences="1">
+      <code>testInvalidPayload</code>
```

Comment:
This seems to be raised because `provideInvalidPayload` returns

`@return iterable<int, list<array<string, mixed>>>`

instead of

`@return iterable<int, array{array<string, mixed>}>`

Psalm 5.3 became a bit more precise around here :)